docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -28,7 +28,7 @@ Before you can specify an equivalent COM add-in, you must first identify its `Pr
 ### Configure the manifest
 
 > [!IMPORTANT]
-> Applies to Excel, Outlook, PowerPoint, and Word.
+> Applies to **Excel**, **Outlook**, **PowerPoint**, and **Word**.
```

Comment:
Curious why we're bolding the applications?

Reply:
It was based on a suggestion to make supported hosts mentioned inline more visible. For an example, see [Create custom contextual tabs in Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/design/contextual-tabs#prerequisites).

Reply:
I see. I guess because it's already in a very short note, it doesn't seem like bolding would add to the visibility

Reply:
That makes sense. I reverted the bold format.